From f3f67b79195cb9dbb6f7c2401ffe17d3e3b8321b Mon Sep 17 00:00:00 2001
From: Jindřich Makovička
Date: Sun, 26 Oct 2014 17:15:49 +0100
Subject: fix copy/paste error

This regression was introduced by commit
6365d18ad0ab6f3646e220bfc493bb2422c6f9aa

diff --git a/src/plugins.c b/src/plugins.c
index 18278fa..55cd157 100644
--- a/src/plugins.c
+++ b/src/plugins.c
@@ -1349,7 +1349,7 @@ replace_plugins()
 			{
 			gtk_box_pack_start(GTK_BOX(gkrellm_monitor_vbox()),
 					mon->privat->main_vbox, FALSE, FALSE, 0);
-			g_object_ref(G_OBJECT(mon->privat->main_vbox));
+			g_object_unref(G_OBJECT(mon->privat->main_vbox));
 			}
 		}
 	}
-- 
cgit v0.10.2-6-g49f6

